js/snippetPreview.js

```diff
@@ -397,3 +673,134 @@ YoastSEO.SnippetPreview.prototype.setFocus = function( ev ) {
 		}
 	}
 };
+
+/**
+ * Binds the reloadSnippetText function to the blur of the snippet inputs.
+ */
+SnippetPreview.prototype.bindEvents = function() {
+	var targetElement,
+		elems = [ "title", "slug", "meta-description" ],
+		focusBindings = [
+			{
+				"click": "title",
+				"focus": "title"
+			},
+			{
+				"click": "urlPath",
+				"focus": "urlPath"
+			},
+			{
+				"click": "urlBase",
+				"focus": "urlPath"
+			},
+			{
+				"click": "metaDesc",
+				"focus": "metaDesc"
+			}
+		];
+
+	for ( var i = 0; i < elems.length; i++ ) {
+		targetElement = document.getElementsByClassName( "js-snippet-editor-" + elems[ i ] );
+		targetElement = targetElement[0];
+
+		targetElement.addEventListener( "keydown", this.changedInput.bind( this ) );
+		targetElement.addEventListener( "keyup", this.changedInput.bind( this ) );
+	}
```

Comment:
It might be necessary to add more events, when cut/pasting with a mouse, (not all) browsers detect this as an input change, so nothing happens when a text is inserted or removed.